src/App.js

```diff
@@ -5,8 +5,8 @@ import { View, StyleSheet, ActivityIndicator, Platform, Keyboard, FlatList } fro
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import ListView from 'deprecated-react-native-listview';
 
-import Header from './components/header';
-import Footer from './components/footer';
+import TodoInput from './components/todoInput';
```

Comment:
Much better name! Nicely done 👍

Reply:
At the time of your comment a i haven't added any commits related to the navigation or the addition of screens, should i wait to have commits related to the PR before opened the PR?